HelpSource/Tutorials/Getting-Started/02-First-Steps.schelp

```diff
@@ -96,45 +96,43 @@ line 3 char 10:
-
-By convention this kind of key sequence is written Cmd - \
+A couple of more notes about the post window. It's very useful to be able to see it, but sometimes it can get hidden behind other windows. You can bring it to the front at any time by pressing Cmd-\.
 
-As well, sometimes the post window becomes full of stuff and hard to read. You can clear it at any time by pressing Cmd-shift-p (hold down the command key and the shift key, and then press p).
+Other times, the post window becomes full of text and is hard to read. You can clear it at any time by pressing Cmd-Shift-P.
 
 section::The World According to SuperCollider
 
 SuperCollider is actually three programs:
 list::
-##The text editor (or IDE) which you are looking at,
-##the language (sclang, or 'client' app),
+##The text editor you are looking at (also referred to as the IDE or Integrated Development Environment),
+##the language (sclang or the "client" app),
 ##and the server, which does the actual synthesis and calculation of audio.
 ::
-The sclang part is a sophisticated programming language with nice GUI features; and the server part is a mean, lean, efficient UNIX command line application (meaning it runs without any GUI representation).
+The sclang part is a sophisticated programming language with nice features for building GUI's (Graphical User Interfaces); and the server part is a lean, mean, efficient UNIX command line application (meaning it runs without any GUI representation).
 
-They communicate by a protocol called Open Sound Control (OSC), over either UDP or TCP, which are network protocols also used on the internet. Don't think from this that the two applications must run on different computers (they can, which can have definite performance advantages), or that they need to be connected to the internet (although it is possible to have clients and servers in different parts of the world communicating!!). Most of the time they will be running on the same machine, and the 'networking' aspect of things will be relatively transparent for you.
+They communicate by a protocol called Open Sound Control (OSC), over either UDP (User Datagram Protocol) or TCP (Transmission Control Protocol), which are network protocols also used on the internet. However, just because these two applications use protocols used on the internet, it is not necessary for them to run on different computers (though they can and there can be performance advantages to doing so), or that they need to be connected to the internet (although it is possible to have clients and servers in different parts of the world communicating!!). Most of the time they will be running on the same computer, and the "networking" aspect of things will be relatively transparent for you.
```

Comment:
would you mind giving this paragraph another look? it doesn't make grammatical sense to me, and gets a bit convoluted with the parenthetical asides. in my experience, technical documentation is easier to read when there are fewer parentheticals. opinions differ, of course. also to be clear the explanations of abbreviations such as "UDP (User Datagram Protocol)" are totally fine.

Reply:
Understood, and I think I agree with you on your point about parentheticals. I'm taking a crack at making it more comprehensible, but feel free to lmk if there's anything that could still be improved in the next revision. This is kind of a cool point and it would be nice to make it sound exciting and clear to veteran coders while not too complex and scary to someone not super familiar with programming